### ISSUE TITLE
Reduced Line Height Is Also Reduced Readability In MonoSpace

### ISSUE THREAD

**Issue by ghost** [mozilla/Fira#189]
Fira Mono line height was 1.4 and reduced 1.2 which also reduced the readability considerable.
The first pic is from an older version, it is crystal clear and readability is perfect.
https://s10.postimg.io/6nxk7cxgn/bandicam_2016_09_09_18_55_01_079.png
The second pic is from a newer version whose characters are so close and seems to be in a somewhat chaos because they are standing only barely below each other.
https://s17.postimg.io/mjf1onhx9/bandicam_2016_09_09_18_55_41_531.png

I dont care about Sans and Serif but please just bring back the line height around 1.333 in MonoSpace version.

**Comment by malaybasu** [comment]
Yes. I strongly agree. The change is horrible. The code font should return back to line-height 1.4. I have switched to "Courier-prime-code", which has extra line-height. Sad, old Fira-Mono was one of the best code fonts.

**Comment by krono** [comment]
The 1.2 lineheight looks consistent. I'd say its a task of the editor to adjust if desired, not the task of the font…

**Comment by malaybasu** [comment]
And pray tell, which programming editor allows you to change the line-height?

**Comment by malaybasu** [comment]
And what about terminals. A monospace font is pervasive in programming world and has to look good out of the box.

**Comment by jfkthame** [comment]
Note that the line-height of Fira Mono has come up for discussion previously, with some people specifically asking for it to be reduced. (See issue #67.) I don't think there's any way to keep everyone happy.... :-(

**Comment by malaybasu** [comment]
@jfkthame I completely understand this. I think the change is trivial. How about creating a version "Fira-Mono-Code" with extra line height. Courier-Prime did it with the creation of "Courier-Prime-Code".

**Comment by malaybasu** [comment]
Looks like Fira code will actually have tighther spacing. Now, can we get back our belove line-spacing please. :-)
